src/install/aim/agent-lang/node-config.mdx

```diff
@@ -0,0 +1,63 @@
+---
+componentType: default
+headingText: Configure the Node agent
+---
+
+You installed a Node agent in the last step, but your agent hasn't been configured to collect AI data like LLM events. This step has you apply those configurations. For AI data, you need to enable the `ai_monitoring` flag.
+
+1. We recommend applying these configurations from `newrelic.ini`. If you have a more complex set up with multiple environments, you can configure with environment variables.
+
+    <CollapserGroup>
+        <Collapser
+            id="config-file"
+            title="Enable through `newrelic.ini`"
+        >
+
+    ```yaml
+    ai_monitoring.enabled = true
+    ai_monitoring.streaming = true
+    ```   
+        </Collapser>
+        <Collapser
+            id="config-enviro"
+            title="Enable with environment variables"
+        >
+
+    ```bash
+    NEW_RELIC_AI_MONITORING_ENABLED=true
+    NEW_RELIC_AI_MONITORING_STREAMING=true
+    ```
+
+        </Collapser>
+    </CollapserGroup>
+
+2. At this point, you've programmed your agent to collect AI data and can start observing your AI app's performance. We recommend you make additional configurations in the `newrelic.ini` file, however:
+
+    <table>
+        <tbody>
+            <tr>
+                <td>
+                Enable distributed tracing
+                </td>
+                <td>
```

Comment:
@RyanKadri Last I heard, the distributed tracing feature flag needs to be enabled for the agent to capture data about every service/db/whathaveyou in a toolchain. This seems like it's a required feature flag. Is this true? Can we move this to step one with the Ai_monitoring feature flag?